lib/webauthn/authenticator_data.rb

```diff
@@ -26,7 +27,11 @@ def valid?
     end
 
     def user_present?
-      flags[USER_PRESENT_FLAG_POSITION] == "1"
+      flags[USER_PRESENT_FLAG_POSITION] == "1" || user_verified?
```

Comment:
This loses the ability to verify only a UP flag. Changing `valid?` method to check both might be better?

Reply:
That's true.

The thing is I wanted to keep this logic contained within `AuthenticatorData`. I think I can get away with both things by reworking a bit this methods in `AuthenticatorData`...

Reply:
Thank you for your feedback! :-)